Library/Homebrew/cmd/fetch.rb

```diff
@@ -64,7 +65,7 @@ def fetch
   def fetch_bottle?(f)
     return true if ARGV.force_bottle? && f.bottle
     return false unless f.bottle && f.pour_bottle?
-    return false if ARGV.build_from_source? || ARGV.build_bottle?
+    return false if FormulaInstaller.build_formula_from_source?(f)
```

Comment:
Ah, gotcha. Do you think `build_formula_from_source?` makes more sense on `ARGV`?